soh/soh/Enhancements/game-interactor/GameInteractor_HookTable.h

```diff
@@ -73,7 +73,10 @@ DEFINE_HOOK(OnUpdateFileBossRushOptionSelection, (uint8_t optionIndex, uint8_t o
 DEFINE_HOOK(OnUpdateFileRandomizerOptionSelection, (uint8_t optionIndex));
 DEFINE_HOOK(OnUpdateFileNameSelection, (int16_t charCode));
 DEFINE_HOOK(OnFileChooseMain, (void* gameState));
+
+// Rando Generation
 DEFINE_HOOK(OnGenerationCompletion, ());
+DEFINE_HOOK(ShouldAddLocationToPool, (Rando::Location * location, bool* result));
```

Comment:
```suggestion
DEFINE_HOOK(ShouldAddLocationToPool, (Rando::Location* location, bool* result));
```
surprised clang-format let this by

Reply:
I had it like that before, clang-format was the one that changed it.